test/compile/selection/point.test.ts

```diff
@@ -186,6 +186,30 @@ describe('Multi Selection', () => {
 
     const signals = assembleUnitSelectionSignals(model, []);
     expect(signals).toEqual(expect.arrayContaining([...oneSg, ...twoSg, ...threeSg, ...fourSg, ...fiveSg, ...sixSg]));
+
+    // Ensure that interval brushes are accounted for.
+    // (We do this separately so as not to pollute other test cases.)
```

Comment:
I'm not following what "separately" means here. Separately from what?

Reply:
Separately defining the selections necessary for this case from the block of selection definitions at the top of this file.

Reply:
```suggestion
    // (We define selections separately so as not to pollute other test cases.)
```